### DISCUSSION TITLE
我應該已經全照做了但什麼都刮削不到

### DISCUSSION THREAD

**Discussion by ITtest1** [metatube-community/jellyfin-plugin-metatube#124] [Q&A]
為什麼我什麼都刮不到嗎？我在jellyfin安裝了插件metatube並且填上了docker安裝的metatube的地址也上了證書，也在媒體庫裡選上metatube做刮削，還有什麼我沒有做嗎？搜了一整天竟然沒有一貼metatube相關的教學，研究了一整晚已經筋疲力盡。有哪個大佬有用metatube拜託指一指路。
jellyfin    | [00:08:34] [INF] [66] Jellyfin.Plugin.MetaTube.Providers.MovieProvider: Search for movie: IPX-622
jellyfin    | [00:08:35] [INF] [66] Jellyfin.Plugin.MetaTube.Providers.MovieProvider: Get movie info: :
jellyfin    | [00:08:35] [ERR] [66] MediaBrowser.Providers.Movies.MovieMetadataService: Error in MetaTube
jellyfin    | System.Exception: API request error: 404 (Not Found)
jellyfin    |    at Jellyfin.Plugin.MetaTube.ApiClient.GetDataAsync[T](String url, Boolean requireAuth, CancellationToken cancellationToken)
jellyfin    |    at Jellyfin.Plugin.MetaTube.ApiClient.GetMovieInfoAsync(String provider, String id, Boolean lazy, CancellationToken cancellationToken)
jellyfin    |    at Jellyfin.Plugin.MetaTube.ApiClient.GetMovieInfoAsync(String provider, String id, CancellationToken cancellationToken)
jellyfin    |    at Jellyfin.Plugin.MetaTube.Providers.MovieProvider.GetMetadata(MovieInfo info, CancellationToken cancellationToken)
jellyfin    |    at MediaBrowser.Providers.Manager.MetadataService`2.ExecuteRemoteProviders(MetadataResult`1 temp, String logName, TIdType id, IEnumerable`1 providers, CancellationToken cancellationToken)
![Screenshot_20230124-074040_Jellyfin](https://user-images.githubusercontent.com/87193498/214180898-75d930ed-ef06-4cac-8f3d-3b99fcf878fd.jpg)
![Screenshot_20230124-074202_Jellyfin](https://user-images.githubusercontent.com/87193498/214180907-59b9a3d0-1b69-4c8c-86ba-fae8f3e7bb9e.jpg)
![Screenshot_20230124-074320_Jellyfin](https://user-images.githubusercontent.com/87193498/214180909-be07a903-2d0c-40d3-98d0-f311519fea64.jpg)
![Screenshot_20230124-074300_Jellyfin](https://user-images.githubusercontent.com/87193498/214180912-1fb58c1c-2f11-49df-8652-eee44276aff1.jpg)
![Screenshot_20230123-213615_VNC Viewer](https://user-images.githubusercontent.com/87193498/214180913-151254d2-5a0d-4fdf-b8f0-e2cf469a918e.jpg)
![Screenshot_20230124-074008_Jellyfin](https://user-images.githubusercontent.com/87193498/214180914-d953662a-ed94-4ef1-b6d6-601bce76f6a4.jpg)

**Comment by iamckun** [comment]
解决了没吧插件的配置截图全一点

**Comment by ITtest1** [comment]
解決了，你是作者嗎？為什麼dldss和fsdss經常刮不到，經常刮錯，明明在jav321有搜索到，手動識別就是沒有

**Comment by zhayo0** [comment]
我也是一样。我还以为就是我自己的问题。正想发帖呢

**Comment by LegendAdmin** [comment]
怎么解决的？我什么都刮削不到，无论是公益服还是自建后端，我用自建后端的时候可以看到clash有javbus，我去javbus搜了一下，明明有对应的资料，但插件却什么都没刮削出来

---

更新：
解决了，见#208

**Comment by xianyunyeheA** [comment]
最好能发下MetaTube后端的日志
